src/app/public/modules/phone-field/types/number-return-format.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Specifies the return format for the validated phone number. TODO: DENISE HELP WITH THE WORDING
```

Comment:
Want to give it a "fair shake" first?

Reply:
Sorry, was eating lunch!

Specifies the format for validated phone numbers using the default country's format. 

I don't like that I used format twice though. I thought I saw @blackbaud-johnly leave a comment, but I'll let him chime in. I also agree with his suggestion of defining the two other options in the JSdocs if we can.

Reply:
Denise and I had already had a call around this and she is working on the messaging. I left the comment in there just to remind myself that this needed finished.

Reply:
@Blackbaud-SteveBrush and @blackbaud-johnly - just to update you on what @Blackbaud-DeniseCisneros and I talked about offline yesterday. Basically we can't use an enum here because this is going to be set in an HTML template and you can't import enums into HTML templates easily without jumping through some hoops. So I went with a `type` here. However, `type` classes don't seem to have a way in `docs-tools` to define the individual values (unless I'm missing something). So the description will end up having to be all together.

@Blackbaud-DeniseCisneros yesterday we had discussed having the first sentence basically describe what the model was for and then a second one describe `reactive` as that was the only one that isn't straighforward via the name. Are we trying to just do one sentence now?

Reply:
Ah, right right. I forgot the bit about the enums until you mentioned it again just now. Do you think the description about the default country's format plus listing the default value as `reactive` would be enough or could it be misunderstood?

_Or_ should we end the description after "validated phone numbers" and start a new sentence providing descriptions for the other values? On second thought as I'm typing this, I think if we're going to document `reactive`, we should also include `national` and international` even if they seem self-explanatory.

Reply:
I agree about describing all the valid options. It seems like we could start with `Specifies the format for validated phone numbers.` and then just have a short description of each option. I know we have other examples in the existing docs where we followed that pattern. As we convert to JSDocs, we're losing those examples because we're pushing the option descriptions to the tables that `docs-tools` creates, but I'm pretty sure we still have examples in the docs somewhere.

Reply:
I've updated both the input description and the type alias description based on some conversations I've had with Denise.